R/load.R

```diff
@@ -454,17 +630,22 @@ findPeers <- function(array, array.list, peer.direction = c("before", "after"),
         # round <- round + 1 # debug counter
         # cat("Round", round, "\n") # debug and testing
 
-        # Check every array that is not the one we're examining and that has not been deemed a peer yet.
-        to.check <- names(array.list)[!(names(array.list) %in% c(array, usable.peers))]
+        # Check every array that is not the one we're examining and that has 
+        # not been deemed a peer yet.
+        to.check <- names(array.list)[!(names(array.list) %in% c(array, 
+                                                                 usable.peers))]
         # cat("Checking:", to.check, "\n") # debug and testing
                         
         check.results <- sapply(to.check, function(x) {
-          # only relevant to test if array x is a valid peer if it connects with anything in the opposite direction.
-          # e.g. if I have A -- B -- C, A cannot be the "after" peer of anyone, because nothing comes "before" it.
+          # only relevant to test if array x is a valid peer if it connects
+          # with anything in the opposite direction.
+          
+          # e.g. if I have A -- B -- C, A cannot be the "after" peer of anyone, 
+          # because nothing comes "before" it.
```

Comment:
```suggestion
          # only relevant to test if array x is a valid peer if it connects
          # with anything in the opposite direction. e.g. if I have 
          # A -- B -- C, A cannot be the "after" peer of anyone, because 
          # nothing comes "before" it.
```